KONDE-AT/dsebaseapp
# Improve modal dialogues

+ modal dialogue for nested rs-tags
+ modal dialogue for rs-tags with more then one reference entry
+ modal dialogue for calendar with more then one letter per day

How to test:
- Open letter "Hermann Bahr an Arthur Schnitzler 19.10.1904" - click on "Schwestern Flöge": A modal dialogue with tree entries should appear (same index: person)
- Open letter "Hugo von Hoffmannsthal an Arthur Schnitzler 30.1.1893 - click on "Russin": A modal dialogue with two entries should appear (different indizes: name, place)
- Open calendar and click on 31.1.1893: A modal dialogue with three entries should appear

A documentation about using the rs tags is included in the base directory: Notes_For_Using_the_rs_Tag.pdf

The code for handling of modals is now in resources/js/custom.js
The modal dialogue for the calendar is handled in resources/js/calendar.custom.js

If you find errors or have improvements, let me know.
